mytoyota/controller.py

```diff
@@ -229,6 +234,8 @@ async def request(  # pylint: disable=too-many-branches
                 raise ToyotaApiError("Servers are overloaded, try again later")
             elif response.status_code == HTTPStatus.SERVICE_UNAVAILABLE:
                 raise ToyotaApiError("Servers are temporarily unavailable")
+            elif response.status_code == HTTPStatus.FORBIDDEN:
+                raise ToyotaActionNotSupported("Action is not supported on this vehicle")
```

Comment:
What if another endpoint returns 401?

Reply:
@DurgNomis-drol: Good question.... I'm not sure. This code change indeed raises a more specialized exception, where in the past this could have been a more generic 'ToyotaApiError' exception. 

Now that I'm thinking about it..... I can also inherit the `ToyotaActionNotSupported` exception from the `ToyotaApiError` exception. The existing code that is still depending on catching the `ToyotaApiError` will still work as it will catch the `ToyotoApiError`.

Reply:
@joro75 Sounds good, I don't expect there to be a problem anyway. This is awesome stuff you guys are doing 🎉